README.md

```diff
@@ -101,6 +101,11 @@ Be aware that Cloud Shell has [several limitations][cloud-shell-limitations],
 in particular an inactivity timeout that will close running shells after 20
 minutes. Please consider it only for blueprints that are quickly deployed.
 
+## VM Image Support
+
+The HPC Toolkit supports the HPC CentOS 7 VM image and the Ubuntu 20.04 LTS
```

Comment:
nit: Does it matter to make the distinction between centos 7 and the hpc centos 7? I might consider listing both.

Reply:
There are changes in the HPC images that may affect how well we support other modules. I'd rather keep this strict for "official" support for now. I am open to further discussion on that though.